patterns/rightsizing-virtual-machines.md

```diff
@@ -0,0 +1,45 @@
+# Rightsizing oversized VMs
+
+## Version
+1.0
+
+## Submitted By
+Bill Johnson
```

Comment:
If we are adding a name, perhaps some links, affiliation, something so we know "which Bill Johnson" is the real "Bill Johnson" (could be the GH id perhaps?)

Reply:
I was planning to use the github id from the PR for this, but we could make it explicit too. I like having that so someone could submit on behalf of if needed